### DISCUSSION TITLE
Is there a way to export cover-art / metadata for use in other media players?

### DISCUSSION THREAD

**Discussion by DSew20** [stashapp/stash#3808] [General]
Firstly, this is brilliant.  Such an easy system for scraping data.

Having got images / metadata for scenes and movies, is there a way to have this available for other players?  Specifically, JellyFin?  I can see great benefit in managing the files in Stash, but playing via JellyFin (or plex) on TV etc.

Any ideas how to make the data / images available to the scraped data can be used across platforms etc?

**Comment by MrX292** [comment]
have not used it, but there is a plugin for JellyFin https://docs.stashapp.cc/add-ons/third-party-integrations/#jellyfinpluginstash there is also one for plex on the same page.
